### PR TITLE
Migrate to Azure Functions v2

### LINKED ISSUE #4: Update Az Functions runtime to 2.0
Let's update the project to the latest version of azure functions so we don't need to run everything on windows 🎉

### PR DESCRIPTION
**Pull request by sakisk**
Closes dotnetsheff/dotnetsheff-api#4

### CODE REVIEW

File: src/dotnetsheff.Api/.gitignore [hunk 1]
```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
```
Comment (kevbite) [inline src/dotnetsheff.Api/.gitignore]: Why do we have another .gitignore file?

File: src/dotnetsheff.Api/KeepAlive/KeepAlive.cs [hunk 2]
```diff
@@ -1,14 +0,0 @@
-using Microsoft.Azure.WebJobs;
```
Comment (kevbite) [inline src/dotnetsheff.Api/KeepAlive/KeepAlive.cs:1]: Why you removed this file?

File: src/dotnetsheff.Api/.gitignore (hunk 1 above)
Comment (sakisk) [reply src/dotnetsheff.Api/.gitignore]: we don't need it. VS added it when i created an new Azure Functions v2 functions. I'll get rid of it

**Review by sakisk — COMMENTED** [review]
I must have missed that one when I was pulling files to new proj. Will add it back

File: src/dotnetsheff.Api/KeepAlive/KeepAlive.cs (hunk 2 above)
Comment (sakisk) [reply src/dotnetsheff.Api/KeepAlive/KeepAlive.cs:1]: Forgot it when moving files on new AzFunc v2 proj. Added it back

**Pull request merged**